src/lib/structures/MoosicClient.ts

```diff
@@ -5,6 +5,17 @@ import Spotify from 'erela.js-spotify';
 import MoosicEmbed from '../extensions/MoosicEmbed';
 import { Logger } from '../util/Logger';
 
+declare module "discord-akairo" {
```

Comment:
This already exists in lib/types/types.d.ts :P

Reply:
mmmmmmm